spanner/spring-data/src/main/com/example/spanner/QuickStartSample.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import com.example.spanner.Singer.Name;
+import java.util.Arrays;
+
+/**
+ * A quick-start sample that uses Spring Data Cloud Spanner to perform read, write, and DDL
+ * operations.
+ */
+public class QuickStartSample {
+
+  public static void main(String[] args) throws InterruptedException {
```

Comment:
If you intend for users to run this, I would suggest adding some print statements to demonstrate to the user that the application is doing something and is not just a no-op.

Even better would be if you made this a REST application using Spring to give some interactivity for the user.

Reply:
Dan, I left those out intentionally actually. The purpose of these examples is to act as snippets for the devsite. I did not want print statements and other junk crowding those snippets.